pkg/gofr/datasource/dgraph/dgraph.go

```diff
@@ -26,7 +26,6 @@ type Config struct {
 // Client represents the Dgraph client with logging and metrics.
 type Client struct {
 	client  DgraphClient
-	conn    *grpc.ClientConn
```

Comment:
Can we get a context of why it existed and now why is it removed ?

Reply:
It existed because we thought of storing the connection inside the `Client` struct but later changed the implementation and used it like this:

```go
conn, err := grpc.Dial(address, grpc.WithTransportCredentials(insecure.NewCredentials()))
if err != nil {
	d.logger.Errorf("error while connecting to Dgraph, err: %v", err)
	return
}
	
dg := dgo.NewDgraphClient(api.NewDgraphClient(conn))
d.client = NewDgraphClient(dg)

```

So this field **conn was unused**, hence removed it.

Reply:
We were doing squash and merge till now but decided to merge all commits from next release